src/yellowhammer/magics.py

```diff
@@ -0,0 +1,86 @@
+"""
+Magics to support LLM interactions in IPython/Jupyter.
+Adapted from fperez/jupytee and jan-janssen/LangSim.
+"""
+import os
+
+from IPython import get_ipython
+from IPython.core.magic import (
+    Magics,
+    magics_class,
+    line_cell_magic,
+)
+from IPython.core.magic_arguments import (
+    magic_arguments,
+    argument,
+    parse_argstring,
+)
+from IPython.display import Markdown
+from .llm import get_chain, code, FinalResponse, ConversationalResponse
+from .prompt import API_PROMPT
+
+
+def get_output(messages, temp=0.1):
+    env = os.environ
+    agent_executor = get_chain(
+        api_provider=env.get("LLM_PROVIDER", "OPENAI"),
+        api_key=env.get("LLM_API_KEY"),
+        api_model=env.get("LLM_MODEL", None),
+        api_temperature=env.get("LLM_TEMP", temp),
+    )
+
+    return agent_executor.invoke({"context": API_PROMPT, "messages": messages})
+
+# Class to manage state and expose the main magics
+@magics_class
+class datalabMagics(Magics):
```

Comment:
same here with camel case

Reply:
Will convert all to camelcase (with the initial cap)